packages/nextjs/package.json

```
@@ -30,11 +30,11 @@
   ],
   "scripts": {
     "prepublishOnly": "npm run build",
-    "build": "tsc -p tsconfig.build.json",
+    "dev": "tsup --watch",
+    "build": "tsup --minify --env.NODE_ENV production",
```

```suggestion
    "build": "tsup --env.NODE_ENV production",
```
I think that minify is enabled by default for prod builds :)